google/cloud/aiplatform/explain/lit.py

```diff
@@ -75,8 +76,109 @@ def spec(self):
         return dict(self._column_types)
 
 
-class _VertexLitModel(lit_model.Model):
-    """LIT model class for the Vertex LIT integration.
+class _EndpointLitModel(lit_model.Model):
+    """LIT model class for the Vertex LIT integration with a model deployed to an endpoint.
+
+    This is used in the create_lit_model function.
+    """
+
+    def __init__(
+        self,
+        endpoint: Union[str, aiplatform.Endpoint],
+        input_types: "OrderedDict[str, lit_types.LitType]",  # noqa: F821
+        output_types: "OrderedDict[str, lit_types.LitType]",  # noqa: F821
+        model_id: Optional[str] = None,
+    ):
+        """Construct a VertexLitModel.
+            Args:
+              model:
+                Required. The name of the Endpoint resource. Format:
+                ``projects/{project}/locations/{location}/endpoints/{endpoint}``
+              input_types:
+                Required. An OrderedDict of string names matching the features of the model
+                as the key, and the associated LitType of the feature.
+              output_types:
+                Required. An OrderedDict of string names matching the labels of the model
+                as the key, and the associated LitType of the label.
+              model_id:
+                Optional. A string of the specific model in the endpoint to create the
+                LIT model from. If this is not set, any usable model in the endpoint is
+                used to create the LIT model.
+            Raises:
+                ValueError if the model_id was not found in the endpoint.
+        """
+        if isinstance(endpoint, str):
+            self._endpoint = aiplatform.Endpoint(endpoint)
+        else:
+            self._endpoint = endpoint
+        self._model_id = model_id
+        self._input_types = input_types
+        self._output_types = output_types
+        # Check if the model with the model ID has explanation enabled
+        if model_id:
+            deployed_model = next(
+                filter(
+                    lambda model: model.id == model_id, self._endpoint.list_models()
+                ),
+                None,
+            )
+            if not deployed_model:
+                raise ValueError(
+                    "A model with id {model_id} was not found in the endpoint {endpoint}.".format(
+                        model_id=model_id, endpoint=endpoint
+                    )
+                )
+            self._explanation_enabled = bool(deployed_model.explanation_spec)
+        # Check if all models in the endpoint have explanation enabled
+        else:
+            self._explanation_enabled = all(
+                model.explanation_spec for model in self._endpoint.list_models()
+            )
+
+    def predict_minibatch(
+        self, inputs: List[lit_types.JsonDict]
+    ) -> List[lit_types.JsonDict]:
+        """Retun predictions based on a batch of inputs.
+            Args:
```

Comment:
It looks like the Args/Returns section need to be deindented one level throughout.